### PR TITLE
Android paywalls: experimental opt-in and version update

### PR DESCRIPTION
**Pull request by NachoSoto**

### CODE REVIEW

File: docs_source/🛠 Tools/paywalls.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ To use RevenueCat Paywalls, simply:
 > 
 > RevenueCat Paywalls are available on the following SDK versions:
 > - iOS: version >= 4.26.0
-> - Android: version >= 7.1.0 (beta)
+> - Android: version >= 7.1.0
```
Comment (tonidero) [inline docs_source/🛠 Tools/paywalls.md:87]: Should we add something here mentioning the apis are experimental but should be mostly production ready?

**Review by tonidero — APPROVED** [review]
Just a comment but we can address in another PR if you prefer

File: docs_source/🛠 Tools/paywalls.md (hunk 1 above)
Comment (NachoSoto) [reply docs_source/🛠 Tools/paywalls.md:87]: Adding that in a separate PR.

Comment (NachoSoto) [reply docs_source/🛠 Tools/paywalls.md:87]: #463

**Pull request merged**